docs/software/devtools/debug/linaro.md

```diff
@@ -0,0 +1,294 @@
+    | Host Name   | `cscsusername@ela.cscs.ch cscsusername@daint.cscs.ch`  |
+    | Remote Installation Directory | `uenv run linaro-forge/24.1.1:/user-tools -- /user-tools/env/forge/` |    
+
+=== "Santis"
+
+    | Field       | Value                                   |
+    | ----------- | --------------------------------------- |
+    | Connection  | `santis`                                |
+    | Host Name   | `cscsusername@ela.cscs.ch cscsusername@santis.cscs.ch`  |
+    | Remote Installation Directory | `uenv run linaro-forge/24.1.1:/user-tools -- /user-tools/env/forge/` |
+    | Private Key | `$HOME/.ssh/cscs-key` |
+
+=== "Clariden"
+
+    | Field       | Value                                   |
+    | ----------- | --------------------------------------- |
+    | Connection  | `clariden`                                |
+    | Host Name   | `cscsusername@ela.cscs.ch cscsusername@clariden.cscs.ch`  |
+    | Remote Installation Directory | `uenv run linaro-forge/24.1.1:/user-tools -- /user-tools/env/forge/` |
+    | Private Key | `$HOME/.ssh/cscs-key` |
+
+Some notes on the examples above:
+
+* SSH Forwarding via `ela.cscs.ch` is used to access the cluster.
+* replace the username `cscsusername` with your CSCS user name that you would normally use to open an SSH connection to CSCS.
+* `Remote Installation Path` is pointing to the install directory of ddt inside the image
+* private keys should be the ones generated for CSCS MFA, and this field does
+  not need to be set if you have added the key to your [SSH agent][ref-ssh-agent].
+
+Once configured, test and save the configuration:
+
+1. check whether the configuration is correct, click `Test Remote Launch`.
+2. Click on `ok` and `close` to save the configuration.
+3. You can now connect by going to `Remote Launch` and choose the `Alps` entry.
+   If the client fails to connect, look at the message, check your ssh
+   configuration and make sure you can ssh without the client.
+
+### Setup the user environment and build the executable
+
+Once the uenv is loaded and activated, the program to debug must be compiled
+with the `-g` (for cpu) and `-G` (for gpu) debugging flags. For example, we can
+build a cuda test with a user environment:
+
+```bash
+> uenv start prgenv-gnu:24.11:v1
+> uenv view default
+> nvcc -c -arch=sm_90 -g -G test_gpu.cu
+> mpicxx -g test_cpu.cpp test_gpu.o -o myexe
+```
+
+### Launch Linaro DDT
+
+To use the DDT client with uenv, it must be launched in `Manual Launch` mode
+(assuming that it is connected to Alps via `Remote Launch`):
+
+=== "on local machine"
+
+    Start DDT, and connect to the target cluster using the drop down menu for Remote Launch.
+
+    Click on Manual launch, set the number of processes to listen to, 
+    then wait for the slurm job to start (see the "on Alps" tab).
+        
+    <img src="https://raw.githubusercontent.com/jgphpc/cornerstone-octree/ddt/scripts/img/ddt/0.png" width="600" />
```

Comment:
I think you should be able to use plain markdown syntax for this: https://www.markdownguide.org/basic-syntax/#images and https://squidfunk.github.io/mkdocs-material/reference/images/#usage. Also, maybe the image could live in this repo?

Reply:
https://www.markdownguide.org/hacks/#image-size
> The Markdown syntax for [images](https://www.markdownguide.org/basic-syntax/#images-1) doesn’t allow you to specify the width and height of images.

Reply:
But the mkdocs markdown allows for it (https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions), e.g.:
```md
![Image title](https://dummyimage.com/600x400/){ width="300" }
```